mantidproject/mantidimaging
# Refactors some logic bits from MIImageView into a presenter

Refactors two functions that don't need to be in the view into a presenter, and adds tests for them. 

To test: 
- Check that loading data doesn't have any errors when visualising a stack
- Check that clicking on the histogram changes to the correct place
- Check that dragging the ROI around is OK, and running a filter with it (e.g. Crop coords) runs OK


Fixes #517
